source/_patterns/02-molecules/vue-widgets/src/vue-cards/components/cards.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div
+    class="row scoped-row"
+  >
+    <div
+      v-for="item in cards"
+      :key="item.phone"
+      :class="[classObject, activeHighlight(item.id)]"
+      @click="[toggle($event,item)]"
+    >
+      <div class="card-body">
+        <h4 class="card-title">{{ item.name }}</h4>
+        <h6 class="card-subtitle mb-2 text-muted">{{ item.id }}</h6>
+        <div class="card-text">
+          <ul>
+            <li>{{ item.phone }}</li>
+            <li>{{ item.website }}</li>
+            <li>{{ item.email }}</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+</template>
+
+<script>
+// import existing PRINTING styles through javascript
+import 'molecules/card';
+
+export default {
+  name: 'Card',
+  props: {
+    cards: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+  },
+  data() {
+    return {
+      active_elements: [],
+      filterArray: [],
+      error: null,
+      state: {
```

Comment:
What is this? There are classes smashed in next to other state, but those classes are per-card, not overall for the whole app.